frontend/src/services/Api.ts

```diff
@@ -1100,6 +1100,7 @@ export function deleteServiceTrafficRouting(
 
 export function deleteServiceTrafficRouting(serviceDetail: ServiceDetailsInfo): Promise<ApiResponse<string>[]>;
 
+//eslint-disable-next-line no-restricted-syntax
```

Comment:
Function overloading cannot be converted to arrow function, but I don't know how to add such use case in the eslint rule. Since there is only one case in Kiali application, eslint disable line is added.